plugins/rikki/heroeslounge/components/updatematch/default.htm

```diff
@@ -11,26 +11,34 @@
 
         <div class="card-body">
             <h4 class="card-title text-center">VS <a href="{{ 'team/view' | page({slug:__SELF__.opp.slug})}}"><strong>{{__SELF__.opp.title |upper}}</strong></a></h4>
-          
 
-                    <div class="alert alert-danger" role="alert" id="deadline{{__SELF__.match.id}}">
-                        Please remember to save (<i class="fa fa-floppy-o" aria-hidden="true" ></i>) after uploading all games.
-                        Results need to be reported before
-                        {{__SELF__.match.tbp| date('d M Y H:i', __SELF__.timezone)}}
-                        ({{__SELF__.timezone}} {{__SELF__.timezoneOffset}}),
-                        otherwise it will result in a tie and render BOTH teams inactive.
-                    </div>
+            <div id="matchtime">
+                {% if __SELF__.match.wbp %}
+                <h3>Scheduled for {{__SELF__.match.wbp | date(__SELF__.datetimeFormat, __SELF__.timezone)}}</h3>
```

Comment:
This is a totally new addition because it was bugging me that I couldn't see schedule times on the team matches page after they were scheduled